docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,10 +367,17 @@ which is a CDI bean and can be ``@Inject``ed.
 
 In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
 
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+[IMPORTANT]
+====
+Using a database to store transaction logs has several benefits, but in comparison with using the file system to store the logs, you might notice a reduction in performance.
+Storing transaction logs into a database is slower, creates bigger overhead, and requires higher maintenance.
+Use this approach when prioritizing consistency over throughput.
```

Comment:
I don't think this sentence is true. Consistency must always be guaranteed in XA transactions.